### PR TITLE
Support ENV for all config options

### PR DESCRIPTION
**Pull request by doomspork**
Howdy @mmozuras!  Following up on the conversation [here](https://github.com/mmozuras/pronto/pull/154#r75576452), this PR enables support for ENV variables on all configuration options.

### CODE REVIEW

File: lib/pronto/config.rb [hunk 1]
```diff
@@ -12,6 +12,7 @@ def initialize(config_hash = ConfigFile.new.to_h)
     end
 
     def consolidate_comments?
+      @config_hash['consolidate_comments'] ||= ENV['CONSOLIDATE_COMMENTS']
```
Comment (doomspork) [inline lib/pronto/config.rb]: Should `ENV` variables take precedence?

**Comment by mknapik** [comment]
How about prefixing each `ENV` variable with `PRONTO_` to avoid conflicts?

**Comment by doomspork** [comment]
@mknapik the existing configuration variables don't rely on a prefix but I think that's a good idea, I'll leave that up to @mmozuras

File: lib/pronto/config.rb (hunk 1 above)
Comment (mmozuras) [inline lib/pronto/config.rb]: @doomspork yes - they do everywhere else 😄

**Comment by mmozuras** [comment]
@mknapik sounds like a good idea. Could you (or @doomspork) make a separate pull request for it?

**Comment by doomspork** [comment]
@mmozuras I can update this PR.  Do you want me to prefix _all_ the configuration options (a breaking change)?

**Comment by mmozuras** [comment]
@doomspork nah, just update according to my one comment. And then let's make a separate PR with _all_ options prefixed.

**Comment by doomspork** [comment]
@mmozuras sounds good! 👍

**Comment by doomspork** [comment]
Updated @mmozuras 👍

**Comment by mmozuras** [comment]
@doomspork thanks! 🙇

**Pull request merged**